README.md

```diff
@@ -877,7 +874,6 @@ This is a 'catch-all' category for anything that doesn't fit well anywhere else.
 [267]: http://blog.pkh.me/p/20-templating-in-c.html
 [269]: https://github.com/AbsInt/CompCert
 [270]: https://github.com/docopt/docopt.c
-[271]: http://marek.vavrusa.com/c/memory/2015/02/20/memory/
```

Comment:
Seems like the URL for this article got changed to http://marek.vavrusa.com/memory/, and is still available here.

Could you update to this link instead of removing this?